### PR TITLE
gcp-test: update check to run instance with confidential type `sev-snp`

### PR DESCRIPTION
**Pull request by HuijingHei**
Based on https://github.com/coreos/coreos-assembler/pull/3871

### CODE REVIEW

**Review by yasminvalim — DISMISSED** [review]
/lgtm

**Comment by dustymabe** [comment]
it feels weird to me that this test is named `nvme-symlink` but runs on confidential VMs. I'd never guess it from the name. Maybe `confidential-vm-nvme-symlink` would work better?

**Review by dustymabe — DISMISSED** [review]
LGTM

**Comment by HuijingHei** [comment]
> it feels weird to me that this test is named `nvme-symlink` but runs on confidential VMs. I'd never guess it from the name. Maybe `confidential-vm-nvme-symlink` would work better?

Updated, thanks!

File: tests/kola/platforms/gcp/confidential-vm-nvme-symlink [hunk 1]
```diff
@@ -8,10 +8,11 @@
 
 # See https://issues.redhat.com/browse/OCPBUGS-7582
 # https://github.com/coreos/fedora-coreos-tracker/issues/1457
+# https://github.com/coreos/coreos-assembler/issues/3556
 #
 # Force this test to not run by default unless named specifically
 # or `--tag confidential` is passed to `kola run`, also requires
-# `--gcp-machinetype n2d-standard-2 --gcp-confidential-vm`
+# `--gcp-machinetype n2d-standard-2 --gcp-confidential-type SEV_SNP`
```
Comment (travier) [inline tests/kola/platforms/gcp/confidential-vm-nvme-symlink]:
```suggestion
# `--gcp-machinetype n2d-standard-2 --gcp-confidential-type sev_snp`
```
Nit but that's what's documented in the kola help. Only change this if another round is needed

**Review by travier — DISMISSED** [review]

File: tests/kola/platforms/gcp/confidential-vm-nvme-symlink (hunk 1 above)
Comment (HuijingHei) [reply tests/kola/platforms/gcp/confidential-vm-nvme-symlink]: Good catch, updated, thanks!

**Review by dustymabe — APPROVED** [review]
LGTM

**Comment by dustymabe** [comment]
when this merges we'll need to cherry-pick it to `testing` so that our next `stable` build will have successful gcp tests.

**Pull request merged**